packages/text-editor/src/react/source-extension-react.tsx

```diff
@@ -101,4 +101,41 @@ export const SourceExtensionReact = SourceExtension.extend<
 	addNodeView() {
 		return ReactNodeViewRenderer(Component);
 	},
+
+	parseHTML() {
+		return [
+			{
+				tag: "span[data-node-id][data-output-id]",
```

Comment:
Process on paste in TextEditor: 

1. called `renderHTML` to paste HTML expression
2. called `parseHTML` for parsing new GisseleNode attributes
3. create new GiselleNode Component